File: src/codeforces/cf1A.py
from math import ceil

def main():
    n, m, a = map(lambda x: float(x), raw_input().split(' '))
    print '%d' % (ceil(n/a) * ceil(m/a))

if __name__ == '__main__':
    try:
        main()
    except:
        pass

File: src/codeforces/cf1A.c
#include <stdlib.h>
#include <stdio.h>
#include <math.h>

int main(int argc, char const *argv[]) {
    double n, m, a, x;


    while (scanf("%d %d %d\n", &n, &m, &a)) {
        printf("%d %d ", ceil(n/a), ceil(m/a));
        x = ceil(n/a) * ceil(m/a);
        printf("%d\n", x);
    }

    return 0;
}
